usr/share/rear/prep/default/310_include_uefi_tools.sh

```diff
@@ -5,10 +5,13 @@ if grep -qw 'noefi' /proc/cmdline; then
     return
 fi
 
-# next step, is checking /boot/efi directory case-insensitive for the /EFI part (we need it)
-if [[ $(find /boot -maxdepth 1 -iname efi -type d | wc -l) -eq 0 ]] ; then
-    return    # must be mounted
-fi
+# Next step, is checking /boot/efi directory case-insensitive for the /EFI part (we need it).
+# If no /boot/[eE][fF][iI] directory can be found we cannot copy the UEFI binaries we might need.
```

Comment:
@jsmeix I still haven't found a reason why there is a need to check for /boot/[eE][fF][iI] in 320_include_uefi_env.sh when the same check has already been done in 310_include_uefi_tools.sh.